### PR TITLE
Dca 203 create bitbucket restore script for deploying to quickstart

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/util/bitbucket/upload_attachments.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+FREE_SPACE_GB=$((${FREE_SPACE_KB}/1024/1024))
+REQUIRED_SPACE_GB=$((5 + ${ARTIFACT_SIZE_GB}))
+if [[ ${FREE_SPACE_GB} -lt ${REQUIRED_SPACE_GB} ]]; then
+   echo "Not enough free space for download."
+   echo "Free space: ${FREE_SPACE_GB} GB"
+   echo "Required space: ${REQUIRED_SPACE_GB} GB"
+   exit 1
+fi;
+
+wget -qO- ${ATTACHMENTS_TAR_URL}| tar -xvz -C ${ATTACHMENT_DIR_DATA} --strip-components 1
```
Comment (ometelytsia) [inline app/util/bitbucket/upload_attachments.sh]: Why do we need `--strip-components 1`?

Comment (ometelytsia) [inline app/util/bitbucket/upload_attachments.sh]: Could you pls add `time` to this command.
This help us to track and improve timings in future if needed.

Comment (ometelytsia) [inline app/util/bitbucket/upload_attachments.sh]: Looks like tar with `-v` output all unarchived file names into the log file.
As we have a huge amount of files, pls check if we could use: --checkpoint=.10000 flag and remove `-v` -> this way tar will print one dot per 10k files.

File: app/util/bitbucket/upload_attachments.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```
Comment (ometelytsia) [inline app/util/bitbucket/upload_attachments.sh]: Can we add a check that we are on NFS server, but not on the node.
There is a big chance that the user could start this script on the backend node.

File: app/util/bitbucket/populate_db.sh [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    echo "Check correctness of install command or install Postgres client manually."
+    echo "INSTALL_PSQL_CMD=${INSTALL_PSQL_CMD}"
+    exit 1
+  fi
+else
+  echo "Postgres client is already installed"
+fi
+
+echo "Step2: Download DB dump"
+DUMP_DIR='/media/atl/bitbucket/shared'
```
[the remark this answers is not in the document]
Comment (SergeyMoroz0703) [reply app/util/bitbucket/populate_db.sh]: There is the place where free disk space is available 
`1.2T  500G  700G  42% /media/atl`

File: app/util/bitbucket/upload_attachments.sh (hunk 1 above)
Comment (SergeyMoroz0703) [reply app/util/bitbucket/upload_attachments.sh]: --strip-components 1 allow us to untar without creation additional directory inside the `data` dir.

Comment (SergeyMoroz0703) [reply app/util/bitbucket/upload_attachments.sh]: Good point, already tested it, will add.

Comment (ometelytsia) [reply app/util/bitbucket/upload_attachments.sh]: why do we have this additional directory?

File: app/util/bitbucket/populate_db.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (ometelytsia) [reply app/util/bitbucket/populate_db.sh]: looks like this folder is on a NFS server

**Pull request closed without merging**